gp-templates/translation-row.php

```diff
@@ -192,23 +205,24 @@
 			<?php endif; ?>
 			</dl>
 
-			<?php $extra_args = $t->translation_status? array( 'filters[translation_id]' => $t->id ) : array(); ?>
 			<dl>
-			<?php
-					$permalink_filters = $t->translation_status ? array( 'filters[status]' => 'either', 'filters[original_id]' => $t->original_id ) : array( 'filters[original_id]' => $t->original_id );
-					$permalink = gp_url_project_locale( $project, $locale->slug, $translation_set->slug,
-						array_merge( $permalink_filters, $extra_args ) );
-					$original_history = gp_url_project_locale( $project, $locale->slug, $translation_set->slug,
-						array_merge( array('filters[status]' => 'either', 'filters[original_id]' => $t->original_id, 'sort[by]' => 'translation_date_added', 'sort[how]' => 'asc' ) ) );
-			?>
 			    <dt><?php _e( 'More links:', 'glotpress' ); ?>
 				<ul>
-				<?php if ( $t->translation_status ) : ?>
-					<li><a tabindex="-1" href="<?php echo $permalink; ?>" title="<?php esc_attr_e( 'Permalink to this translation', 'glotpress' ); ?>"><?php _e( 'Permalink to this translation', 'glotpress' ); ?></a></li>
-				<?php else : ?>
-					<li><a tabindex="-1" href="<?php echo $permalink; ?>" title="<?php esc_attr_e( 'Permalink to this original', 'glotpress' ); ?>"><?php _e( 'Permalink to this original', 'glotpress' ); ?></a></li>
-				<?php endif; ?>
-					<li><a tabindex="-1" href="<?php echo $original_history; ?>" title="<?php esc_attr_e( 'Link to the history of translations of this original', 'glotpress' ); ?>"><?php _e( 'All translations of this original', 'glotpress' ); ?></a></li>
+				    <?php
+				    /**
+				     * Allows to modify the more links in the translation editor.
+				     *
+				     * @since 2.3
+				     *
+				     * @param array $more_links The links to be output.
+				     * @param GP_Project $project Project object.
+				     * @param GP_Locale $locale Locale object.
+				     * @param GP_Translation_Set $translation_set Translation Set object.
+				     * @param GP_Translation    $t    Translation object.
+				     */
+					foreach ( apply_filters( 'gp_translation_row_template_more_links', $more_links, $project, $locale, $translation_set, $t ) as $link => $title ) : ?>
```

Comment:
* Code indentation looks a bit off here.
* The `@param` tags should be aligned.
* The `title` attribute should be removed. #a11y